MonitorControl/Support/AppDelegate.swift

```diff
@@ -71,11 +71,6 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     }
     prefs.set(currentBuildNumber, forKey: PrefKey.buildNumber.rawValue)
     self.setDefaultPrefs()
-    if !DEBUG_MACOS10, #available(macOS 11.0, *) {
-      self.statusItem.button?.image = NSImage(systemSymbolName: "sun.max", accessibilityDescription: "MonitorControl")
-    } else {
-      self.statusItem.button?.image = NSImage(named: "status")
-    }
```

Comment:
Moved to `MenuHandler.swift` to update Menu Bar Icon when user changes in preferences panel.